Resources/views/Field/preview/content.html.twig

```diff
@@ -0,0 +1,10 @@
+{% if value is not null %}
+    <em>{{ value.name }}</em>
+    <hr>
+    {% for fieldValue in value.fieldValues %}
+        {{ acb_field_preview(fieldValue) }}
+        {% if not loop.last %}
+            <hr>
+        {% endif %}
+    {% endfor %}
+{% endif %}
```

Comment:
infinite loop if the selected content is the parent of the field
(same for front template)